web/src/agama.ts

```diff
@@ -104,8 +104,15 @@ const agama = {
    * @param options passed to the Intl.ListFormat constructor
    */
   formatList: (list: string[], options: object): string => {
-    const formatter = new Intl.ListFormat(agama.language, options);
-    return formatter.format(list);
+    try {
+      // convert the locale string ("_" separator) to HTTP language code ("-" separator)
```

Comment:
Sorry, WHAT? Are we adding the same bug here that we are fixing in other parts of this PR??